service/src/main/java/tds/assessment/repositories/impl/ItemQueryRepositoryImpl.java

```diff
@@ -75,10 +140,13 @@ public ItemQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
             "   propdescription AS description \n" +
             "FROM \n" +
             "   itembank.tblitemprops P \n" +
-            "JOIN itembank.tblsetofadminsubjects segments \n" +
-            "    ON segments._key = P._fk_adminsubject \n" +
+            "JOIN " +
+            "   itembank.tblsetofadminsubjects segments \n" +
+            "   ON segments._key = P._fk_adminsubject \n" +
             "WHERE \n" +
-            "   segments.virtualtest = :key OR segments._key = :key AND" +
+            "   segments.virtualtest = :key \n" +
+            "   OR segments._key = :key \n" +
+            "AND\n" +
```

Comment:
Currently without an parenthesis this will get interpreted as 
    `segments.virtualtest = :key OR (segments._key = :key AND isActive = 1)` 
while I think the actual intent is 
    `(segments.virtualtest = :key OR segment._key = :key) AND isActive = 1`

Also, can you add the alias to the `isActive` since I can't tell which table that is from by just looking at this query.